packages/manager/apps/pci-public-ip/src/api/data/import-ip.ts

```diff
@@ -0,0 +1,34 @@
+import { v6 } from '@ovh-ux/manager-core-api';
+import { isEmpty, isObject } from 'lodash';
+import { ImportsIP } from '@/interface';
+
+export const getImportsIPs = async (
+  projectId: string,
+): Promise<ImportsIP[]> => {
+  const { data } = await v6.get(`/ip?type=failover`);
```

Comment:
i think you can use an iceberg fetch to expand all the ips in one call

Reply:
Awesome, a fast and cleaner way :)